<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>852</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>852</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>399</width>
      <height>862</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bg1/Feature Page.png);
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>124</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>212</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>302</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>389</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>479</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>565</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>653</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>741</y>
        <width>357</width>
        <height>83</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton2">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>38</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bg1/Feature Page-1.png);
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>36</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/bg1/Feature Page-2.png);
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>38</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-3.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>37</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-6.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>39</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-7.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>43</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-8.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>39</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-5.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>36</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>396</width>
        <height>852</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;
border-image: url(:/bg1/Feature Page-4.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>40</y>
        <width>71</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background:none;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/caloriesrc.qrc"/>
 </resources>
 <connections/>
</ui>
